storagext/lib/src/types/market.rs

```diff
@@ -7,7 +7,11 @@ use crate::{
     runtime::{
         bounded_vec::IntoBoundedByteVec,
         runtime_types::pallet_market::{
-            deal_parameters::{OffchainDealDurationBound, OffchainDealParameters},
+            deal_parameters::{
+                DealParameters as PolkaStorageDealParameters,
+                OffchainDealDurationBound as PolkaStorageOffchainDealDurationBound,
+                OffchainDealParameters as PolkaStorageOffchainDealParameters,
+            },
```

Comment:
Similarly to the `Sxt` comment, please review the usage of the types and ensure that it matches it's friends.

From the other imports, it seems that instead of `PolkaStorage` these should be prefixed with `Runtime`